branca/element.py

```diff
@@ -88,6 +89,14 @@ def __setstate__(self, state: dict):
 
         self.__dict__.update(state)
 
+    def clone(self):
+        """creates a new copy of an element, but with a unique identifier
+        and without the prior version's relationship to a parent object"""
+        clone = copy(self)
+        clone._id = hexlify(urandom(16)).decode()
```

Comment:
we should split this logic off into a `_create_id` method or something, since we use it now in two places. But okay if you don't do that in this PR.